src/coreclr/tools/aot/ILCompiler/ILCompiler_crossarch.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I don't understand the purpose of the crossarch project

Reply:
This builds a version of ILC that is x64 hosted, even when we otherwise target arm or arm64. It matches what ../crossgen2/crossgen2_crossarch.csproj does. When we're crosstargeting to arm64, we wouldn't be able to run the ilc that we just built because it's for arm64 but the build machine is x64.

The ugly hardcoding of x64 matches what crossgen2 does - it captures the reality that when we're crossbuilding to arm64, the host is x64. I guess I could use HostArch, but this is a copy paste from crossgen2_crossarch.csproj.